src/translations/nbTranslations.json

```diff
@@ -1912,7 +1914,7 @@
         "remove_relation": "Fjern relasjon",
         "remove_relation_confirm_text": "Ønsker du å fjerne denne relasjonen?",
         "search_for_related_dmps": "Søk etter relaterte datahåndteringsplaner",
-        "search_for_related_registrations": "Søk etter relaterte registreringer",
+        "search_for_related_registrations": "Søk etter resultat med bruk av tittel eller DOI",
```

Comment:
Man kan søke etter flere ting enn bare tittel og DOI, så denne ble kanskje mer upresis IMO, men det får være opp til dere å bestemme 😇 
(denne nøkkelen brukes for andre typer også ser jeg)